development/includes/_hover-card.pug

```diff
@@ -63,7 +63,7 @@
   ]
 
 
-nav.blog-preview-list
+nav.hover-card
```

Comment:
why is this isn the change for the hero? Did my 15 min talk not sink in? Please don't include stuff that has nothing to do with the hero. This is the hero-update branch  or did I miss something?